packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -113,18 +118,18 @@ curl -v -X GET \
     "profile": {
         "displayName": "Example Device name",
         "platform": "WINDOWS",
-        "manufacturer": null,
-        "model": null,
-        "osVersion": null,
-        "serialNumber": "Example",
-        "imei": null,
-        "meid": null,
-        "udid": null,
+        "serialNumber": "XXDDRFCFRGF3M8MD6D",
         "sid": "S-1-11-111",
-        "tpmPublicKeyHash":null,
         "registered":true,
         "secureHardwarePresent":false
     },
+    "resourceType": "UDDevice",
+    "resourceDisplayName": {
+    "value": "Example Device name",
+      "sensitive": false
+    },
+    "resourceAlternateId": null,
+    "resourceId": "guo1j774nHEZFHsL10w5",
```

Comment:
Actually who does `resource..` attributes not show up for `Users` Api? Or at least it is not doc'd

Reply:
This is an artifact of `Device` object extending `Resource` class.